frontend/src/metabase/query_builder/components/QueryDownloadWidget.jsx

```diff
@@ -40,22 +45,21 @@ const QueryDownloadWidget = ({
     triggerClasses={cx(className, "text-brand-hover")}
     triggerClassesClose={classNameClose}
   >
-    <Box
-      p={2}
-      width={result.data && result.data.rows_truncated != null ? 300 : 260}
+    <WidgetRoot
+      isExpanded={result.data && result.data.rows_truncated != null ? 300 : 260}
```

Comment:
`WidgetRoot` sets `width` depending on `isExpanded`, but the ternary passes a numeric value here, so it's going to be `true` all the time